.gitignore

```diff
@@ -0,0 +1,47 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# Virtual environment
+venv/
+
+# pip
+*.log
+pip-log.txt
+
+# dotenv environment variables file
+.env
+
+# GitHub folder (ignore unless using workflows)
+.github/
```

Comment:
we use this folder for code owners  and implementation of workflows

Reply:
please fix this

Reply:
sorry I fixed it but I forget to commit it .